open-metadata-implementation/access-services/governance-engine/governance-engine-api/src/main/java/org/odpi/openmetadata/accessservices/governanceengine/api/MetadataElementInterface.java

```diff
@@ -493,6 +493,25 @@ String createRelatedElementsInStore(String            userId,
                                                                                 PropertyServerException;
 
 
+    /**
+     * Link elements as peer duplicates. Create a simple relationship between two elements.
+     * If the relationship already exists, the properties are updated.
+     *
+     * @param userId caller's userId
+     * @param metadataElement1GUID unique identifier of the metadata element at end 1 of the relationship
+     * @param metadataElement2GUID unique identifier of the metadata element at end 2 of the relationship
+     * @throws InvalidParameterException the unique identifier's of the metadata elements are null or invalid in some way; the properties are
+     *                                    not valid for this type of relationship
+     * @throws UserNotAuthorizedException the governance action service is not authorized to create this type of relationship
+     * @throws PropertyServerException there is a problem with the metadata store
+     */
+    void linkElementsAsPeerDuplicates(String userId,
```

Comment:
Why are the properties (eg steward) not included in the request?

Reply:
@mandy-chessell Does it look like you thought about the changes? :)